packages/apps/shopify-app-remix/src/server/authenticate/admin/helpers/trigger-after-auth-hook.ts

```diff
@@ -23,6 +25,7 @@ export async function triggerAfterAuthHook<
         params,
         authStrategy.handleClientError(request),
       ),
+      redirect: config.isEmbeddedApp ? redirectFactory(params, request) : remixRedirect,
```

Comment:
This always passes a `redirect` function to `afterAuth`.
I tried to make the type similar to the `EmbeddedAdminContext` and `NonEmbeddedAdminContext` returned from `authenticate.admin()`, so that `redirect` is only available for embedded apps. But this would require lots of other type changes to the `AppConfig` to make the generics work. 

Alternatively, `redirect` could also be `undefined` for non-embedded apps.